string generator_id
int32 ID
string type
time generation_date
string gen_time
time completed_date
string compl_time
string[] route
string split_attribute1
float32 split1
string split_attribute2
float32 split2
string split_attribute3
float32 split3

string attribute2
float32 value2
string attribute3
float32 value3


bool last_event
bool first_event
